src/documentation/markdown/2017/static-dataset-faq.md

```diff
@@ -1,30 +1,30 @@
-## Static Dataset FAQ
+# Static Dataset FAQ
 
-### What is the difference between the static datasets?
+## What is the difference between the static datasets?
 The primary difference between the datasets is the time period when the data is captured. Each time period contains the most up-to-date data _at the moment in time when the dataset was created_.
 
 |Dataset|Time Period|
 |---|---|
-|Snapshot | 1 month after the timely filing deadline |
+|Snapshot | Typically 1 month |
```

Comment:
```suggestion
|Snapshot | Typically 1 month after the timely filing deadline|
```

Reply:
From her comment, the goal is to avoid the hard deadline of "1 month".  By itself, "Typically 1 month" does not provide enough context to be meaningful. So let's add back the "after the timely..." bit. 
 
We should probably also change that column's header to better reflect it's content:
`Time Period` => `Time of Capture`?

Reply:
Sounds good. I have reverted the change to the original. I agree `Time of Capture` sounds better, I have updated the header for the table.

Changes are included from 2017-2023.